tidy3d/components/base.py

```diff
@@ -563,7 +574,12 @@ def to_hdf5(self, fname: str, custom_encoders: List[Callable] = None) -> None:
 
         with h5py.File(fname, "w") as f_handle:
 
-            f_handle[JSON_TAG] = self._json_string
+            json_str = self._json_string
+            for ind in range(ceil(len(json_str) / MAX_STRING_LENGTH)):
+                key = f"{JSON_TAG}_{ind}" if ind > 0 else JSON_TAG
```

Comment:
So this creates `JSON_STRING, JSON_STRING_1, ...`?
I'm curious if it will make the logic simpler to just not perform any special handling for `ind=0`, like `JSON_STRING_0, JSON_STRING_1, ...`. If the string is short enough to fit in `JSON_STRING_0`, we just leave it named like that. Will it break backwards compatibility with script loading maybe?

Reply:
anyway, see comment and suggestion above, I think it makes things a bit simpler to keep the handling how it is but just have a single function to get the key, treating ind=0 differently.

Reply:
Sounds good, yeah not appending `ind=0` is for backwards compatibility.